README.md

```diff
@@ -136,10 +136,58 @@ During an Uprising, the active player must perform a double hammer flip followed
+
+### Penalties
+
+If a player strikes an illegal nail, a penalty will apply. There are several scenarios with different punishments listed in the following section. Penalties may be incurred if the illegal nail is touched at all by the hammer during the strike. This includes scenarios where the active player strikes both their own nail and the illegal nail.
+
+If a player accidentally strikes a nail without performing the necessary flips and meeting any other requirements or strikes a nail more times than they are allowed, that player must skip their next turn in addition to any punishments earned if those strikes were against an illegal nail. If the offense was particularly egregious, or if extra strikes were intentionally made, all other players may come to a majority agreement as to whether extra punishments are warranted.
+
+If an illegal nail is struck multiple times, a punishment is given for each strike. The player who chooses these punishments may choose a different one each time.
+
+Note that all of these penalties occur during the offending (active) player's turn. For instance, if the active Player is a Lad they may become a Lord as a result of these actions, but the victim of the offense would still have to wait until their turn to Ascend if one of these actions finishes their nail.
+
+Certain penalties can potentially be coordinated by players to achieve a mutual benefit. This is fair game, but the verbal contract is in no way binding, and a player may choose to betray their co-conspirator at the last moment.
+
+#### A Lad strikes another Lad's nail
+
+If a Lad strikes another Lad's nail, the Lad whose nail was struck chooses one of the following actions:
+
+* Do nothing
+* Reset their own nail
+* Swap nails with the Lad who struck the nail
+* Make the Lad who struck the nail drink 3
+* Get one free strike at that Lad's nail before play continues (at least one flip must still be performed)
+
+#### A Lad strikes a center nail
+
+If a Lad strikes one of the center nails and there is at least one Lord, the next Lord in turn order may chooses one of the following actions:
+
+* Do nothing
+* Make the Lad drink 3
+* Reset that Lad's nail
+* Get a free strike at that Lad's nail before play continues
+
+If there are no active Lords, the Lad must reset their nail.
+
+If a Lad strikes the final center nail and finishes it, that player is removed for the remainder of the game and that center nail is reset. This happens instead of any other punishments for that strike.
+
+#### A Lord strikes a Lad's nail
+
+If a Lord strikes a Lad's nail, the Lad may choose one of the following actions:
+
+* Do nothing
+* Get a free strike at the Uprising nail before play continues (the extra flip for striking the nail does not need to be performed)
```

Comment:
If there is no `Uprising` nail, the options are either Do Nothing or reset their nail?

Reply:
Correct. The one situation this gets iffy is if a Lad's nail is halfway down or so and a lord decides to strike it to attempt to bend it. The lad can reset the nail but now they've lost their progress. 

My thinking here is this is fine because a Lord has to spend an entire turn doing this, and there's no way they can keep this up against all lads, especially considering there's no way to cheat the uprising nail as a lord

Reply:
I also kind of like this gives lords a way to target and try to oppress a specific lad